.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
more on .editorconfig [here ](https://blog.theodo.com/2019/08/why-you-should-use-eslint-prettier-and-editorconfig-together/#gatsby-focus-wrapper:~:text=.-,ESLint%2C%20Prettier%20and%20EditorConfig,EditorConfig%2C%20on%20the%20other%20hand%2C%20does%20not%20lint%20or%20format%20your%20code.%20It%20merely%20defines%20a%20standard%20code%20formatting%20style%20guide%20among%20all%20the%20IDEs%20and%20editors%20used%20within%20a%20team%20of%20developers.%20For%20instance%2C%20if%20a%20team%20uses%20two%20main%20editors%20such%20as%20Sublime%20Text%20and%20Visual%20Studio%20Code%2C%20EditorConfig%20allows%20them%20to%20define%20a%20common%20indentation%20pattern%20(spaces%20or%20tabs)%20within%20a%20single%20file.,-Customize%20the%20rules)